virlglrenderer: Remove hardcoded "-fsanitize=address"

"-fsanitize=address" which pulls in asan does not
work with other sanitizers like msan/tsan etc.
(https://crbug.com/898289)

diff --git a/vtest/meson.build b/vtest/meson.build
index 754b8d9..d80cd35 100644
--- a/vtest/meson.build
+++ b/vtest/meson.build
@@ -52,8 +52,8 @@ if with_fuzzer
    vtest_fuzzer = executable(
       'vtest_fuzzer',
       'vtest_fuzzer.c',
-      c_args : [ '-fsanitize=address', '-fsanitize=fuzzer' ],
-      link_args : [ '-fsanitize=address', '-fsanitize=fuzzer' ],
+      c_args : [ '-fsanitize=fuzzer' ],
+      link_args : [ '-fsanitize=fuzzer' ],
       objects : vtest_obj,
       dependencies : libvirglrenderer_dep
    )
diff --git a/tests/fuzzer/meson.build b/tests/fuzzer/meson.build
index 6c8f2df..8ac4f4d 100644
--- a/tests/fuzzer/meson.build
+++ b/tests/fuzzer/meson.build
@@ -27,7 +27,7 @@ assert(cc.has_argument('-fsanitize=fuzzer'),
 virgl_fuzzer = executable(
    'virgl_fuzzer',
    'virgl_fuzzer.c',
-   c_args : [ '-fsanitize=address', '-fsanitize=fuzzer' ],
-   link_args : [ '-fsanitize=address', '-fsanitize=fuzzer' ],
+   c_args : [ '-fsanitize=fuzzer' ],
+   link_args : [ '-fsanitize=fuzzer' ],
    dependencies : [libvirglrenderer_dep, epoxy_dep]
 )
